cmd/snappy-cloud-image/main.go

```diff
@@ -41,7 +41,7 @@ func main() {
 
 	imgDataOrigin := si.NewClient(httpClient)
 	imgDataTarget := cloud.NewClient(cliExecutor)
-	imgDriver := image.NewUDF(cliExecutor)
+	imgDriver := image.NewUDFQcow2(cliExecutor)
```

Comment:
wow, this is a hard one. cow are also initials :)
Your option looks better than NewUDFQCOW2, though.